### ISSUE TITLE
homebrew/php/php56-mcrypt 5.6.10 problem

### ISSUE THREAD

**Issue by DsXack** [Homebrew/homebrew-php#1785]
```sh
brew upgrade               
==> Upgrading 1 outdated package, with result:
homebrew/php/php56-mcrypt 5.6.10
==> Upgrading homebrew/php/php56-mcrypt
==> Downloading https://homebrew.bintray.com/bottles-php/php56-mcrypt-5.6.10.yosemite.bottle.tar.gz

curl: (22) The requested URL returned error: 404 Not Found
Error: Failed to download resource "php56-mcrypt"
Download failed: https://homebrew.bintray.com/bottles-php/php56-mcrypt-5.6.10.yosemite.bottle.tar.gz
Warning: Bottle installation failed: building from source.
==> Downloading https://www.php.net/get/php-5.6.10.tar.bz2/from/this/mirror

curl: (60) SSL certificate problem: Invalid certificate chain
More details here: http://curl.haxx.se/docs/sslcerts.html

curl performs SSL certificate verification by default, using a "bundle"
 of Certificate Authority (CA) public keys (CA certs). If the default
 bundle file isn't adequate, you can specify an alternate file
 using the --cacert option.
If this HTTPS server uses a certificate signed by a CA represented in
 the bundle, the certificate verification probably failed due to a
 problem with the certificate (it might be expired, or the name might
 not match the domain name in the URL).
If you'd like to turn off curl's verification of the certificate, use
 the -k (or --insecure) option.
Error: Failed to download resource "php56-mcrypt"
Download failed: https://www.php.net/get/php-5.6.10.tar.bz2/from/this/mirror

```

**Comment by DsXack** [comment]
Maybe duplicate #1786

**Comment by alanthing** [comment]
Latter half of the error resolved in #1786. First half is being resolved currently as the built bot is building new bottles.

**Comment by alanthing** [comment]
The latest version will be pushed sometime today.
